crates/zksync/core/src/vm/storage_view.rs

```diff
@@ -76,22 +76,22 @@ impl<S: ReadStorage + fmt::Debug> ReadStorage for StorageView<S> {
         if key.address() == &ACCOUNT_CODE_STORAGE_ADDRESS && key.key() == &self.caller.to_h256() {
             let value = StorageValue::zero();
             tracing::trace!(
-                "override read value {:?} {:?} ({:?}/{:?})",
-                key.hashed_key(),
-                value,
-                key.address(),
-                key.key()
+                hash = ?key.hashed_key(),
```

Comment:
`hashed_key`. Honestly this can be removed as it's usually of no importance to us.